templates/vue/src/components/NodeModal.vue

```diff
@@ -560,6 +581,15 @@ export default {
       })
       return ordered
     },
+    videoUrlYoutubeID() {
+      return this.videoUrlEntered
+        ? Helpers.getYoutubeID(this.node.typeData.mediaURL)
+        : ""
+    },
+    accessSubmit() {
+      // Locks access to submit button while youtube video loads to grab duration
+      return this.videoUrlYoutubeID === "" ? true : this.youtubeLoaded
```

Comment:
It would be more readable if you return `this.videoUrlYoutubeID === "" || this.youtubeLoaded` instead. Also, I think this may be what's causing what Narendra mentioned where the loading button is always present since you are only checking for YouTube.